editor/scene_tree_dock.cpp

```diff
@@ -1610,30 +1617,36 @@ void SceneTreeDock::_script_created(Ref<Script> p_script) {
 	_update_script_button();
 }
 
-void SceneTreeDock::_toggle_editable_children() {
+void SceneTreeDock::_toggle_editable_children_from_selection() {
+
 	List<Node *> selection = editor_selection->get_selected_node_list();
 	List<Node *>::Element *e = selection.front();
+
 	if (e) {
-		Node *node = e->get();
-		if (node) {
-			bool editable = EditorNode::get_singleton()->get_edited_scene()->is_editable_instance(node);
+		_toggle_editable_children(e->get(), true);
+	}
+}
 
+void SceneTreeDock::_toggle_editable_children(Node *p_node, bool p_update_popup) {
+
+	if (p_node) {
+		bool editable = !EditorNode::get_singleton()->get_edited_scene()->is_editable_instance(p_node);
+		EditorNode::get_singleton()->get_edited_scene()->set_editable_instance(p_node, editable);
+		if (editable)
+			p_node->set_scene_instance_load_placeholder(false);
+
+		if (p_update_popup) {
```

Comment:
As per your comment

>  AFAIK it's impossible to toggle editable children without closing the menu and it's rebuilt on each reopen, so updating it is useless.

it seems the whole `p_update_popup` logic might not be needed? Did you try it to confirm?

Reply:
I tested it and seems ok. It would break if there was some way to change editable children without closing the popup menu, but I don't think there is one.